peps/pep-0775.rst

```diff
@@ -0,0 +1,136 @@
+PEP: 775
+Title: Make zlib required to build Python
+Author: Stan Ulbrych <stanulbrych@gmail.com>,
+        Petr Viktorin <encukou@gmail.com>
+Discussions-To: https://discuss.python.org/t/lets-make-zlib-required-rather-than-optional-to-build-cpython/23062
+Status: Draft
+Type: Standards Track
+Created: 20-Feb-2025
+Python-Version: 3.14
+Post-History: `23-Jan-2023 <https://discuss.python.org/t/23062>`__
+
+Abstract
+========
+
+Building CPython on systems without the `zlib <https://zlib.net>`_ compression library, except WASI,
+is no longer supported.
+
+The ``zlib`` module is made a required part of the standard library,
+except on WASI.
+
+Builds without ``zlib`` will still be possible, but formally unsupported.
+
+
+Motivation
+==========
+
+The ``zlib`` library, which powers the ``zlib`` Python module,
+is available on virtually all systems.
+
+Many wheels on PyPI, including the `pip`_ installer, require ``zlib``.
+Users of *pip* would consider CPython without ``zlib`` to be broken,
+but mostly don't notice because all major builds of CPython include ``zlib``.
+
+.. _pip: https://pypi.org/project/pip/
```

Comment:
```suggestion
Many wheels on PyPI, including the :pypi:`pip` installer, require zlib.
Users of pip would consider CPython without zlib to be broken,
but mostly don't notice because all major builds of CPython include zlib.
```